tests/integration/states/archive.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+'''
+Tests for the file state
+'''
+# Import python libs
+from __future__ import absolute_import
+import os
+import shutil
+
+# Import Salt Testing libs
+from salttesting.helpers import ensure_in_syspath
+ensure_in_syspath('../../')
+
+# Import salt libs
+import integration
+import salt.utils
+
+STATE_DIR = os.path.join(integration.FILES, 'file', 'base')
+ARCHIVE_DIR = '/tmp/archive/'
```

Comment:
This is going to break on Windows. We need to use attributes of the integration module to construct a platform-agnostic directory.